billy-france/src/test/java/com/premiumminds/billy/france/test/services/export/TestFRCreditReceiptPDFTransformer.java

```diff
@@ -82,7 +82,7 @@ public void setUp() throws FileNotFoundException {
     }
 
     @Test
-    public void testPDFcreation() throws NoSuchAlgorithmException, ExportServiceException, URISyntaxException,
+    public void testPdfCreation() throws ExportServiceException,
```

Comment:
You replaced "PDF" with "Pdf" in some places but not all of them (for example line 118 of this same file). Please make sure that, if you change the naming, it's consistent across the code